### PR TITLE
CARES Act Recovery Rebate Credits

### PR DESCRIPTION
**Pull request by amshoulders**
[Work in progress]

This pull request seeks to resolve Issue #2543 relating to how to add the Recovery Rebate Credits in the CARES Act.  Thus far I have adjusted the `caclfunctions.PersonalTaxCredit()` function and attempted to add relevant parameters to the `policy_current_law.json` file (though I'm not sure I did this correctly and I know I'm missing a few pieces of information).  The parameters I have added/edited are as follows:

`RRS_ps` has been edited for 2020 to include the phase-out starting rates (150,000 for married filing jointly, 112,500 for head of household, and 75,000 for all others)
`RRS_prt` has been added to represent the phase-out rate of 0.05
`RRC_c_unit` has been added to represent the maximum credit amount for filers (2400 for married filing jointly, 1200 for all other adults)
`RRC_c_kids` has been added to represent the credit amount per child (500)

@jdebacker

### CODE REVIEW

**Comment by amshoulders** [comment]
I'm having some difficulty getting the associated test to pass in scenarios where the ARPA credit is in effect but the filer's AGI is between the phaseout start and phaseout end.  This means that either something is going wrong in the `elif` portion of the function, or (more likely) I'm doing something wrong when I calculate the expected values in excel.  @jdebacker would you mind taking a look at what I'm doing and seeing if I'm doing something obviously wrong?  The excel that I've been using is attached - I have one tab for when ARPA is in effect and one tab for when the CARES act is in effect.  I highlighted the columns that are not passing.
[PersonalTaxCreditTest.xlsx](https://github.com/PSLmodels/Tax-Calculator/files/6358156/PersonalTaxCreditTest.xlsx)

File: taxcalc/calcfunctions.py [hunk 1]
```diff
@@ -2562,15 +2582,16 @@ def PersonalTaxCredit(MARS, c00100, XTOT,
...
     if c00100 < RRC_ps[MARS - 1]:
         recovery_rebate_credit = RRC_c * XTOT
-    elif c00100 < RRC_pe[MARS - 1]:
-        prt = ((c00100 - RRC_ps[MARS - 1]) /
+        recovery_rebate_credit += RRC_c_unit[MARS-1] + RRC_c_kids * nu18
+    elif c00100 < RRC_pe[MARS - 1] & c00100 > 0:
+        prt = ((c00100 - RRC_ps[MARS - 1])/
                (RRC_pe[MARS - 1] - RRC_ps[MARS - 1]))
-        recovery_rebate_credit = RRC_c * XTOT * (1 - prt)
+        recovery_rebate_credit = RRC_c * XTOT * (1-prt)
```
Comment (jdebacker) [inline taxcalc/calcfunctions.py:2592]: Have spaces around operators.

**Comment by jdebacker** [comment]
@amshoulders I used your tests with `tuple1` and `tuple2` (slightly modified to drop new arguments) to test the `calcfunctions.PersonalTaxCredit` function on the master branch.  Both those tests passed.  So I think there's something off in your logic, but I can't spot it.  Maybe just focus on the `tuple2` test and see if you can print some intermediate values/see which if statements are triggered.  That might help identify the issue.

**Comment by jdebacker** [comment]
@amshoulders This PR is looking very good - thanks!

I checked out your branch and did a revenue estimate with the PUF.  

Here's Tax-Calculator with the PUF by calendar year:
|         |    2020 |   2021 |   2022 |   2023 |   2024 |   2025 |   2026 |   2027 |   2028 |   2029 |   2030 |   2021-2030 |
|:--------|--------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|------------:|
| IIT     | -291.78 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |     -291.78 |
| Payroll |    0    |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |        0    |

Versus the [JCT estimate](https://www.jct.gov/publications/2020/jcx-12r-20/) by fiscal year:
|         |    2020 |   2021 |   2022 |   2023 |   2024 |   2025 |   2026 |   2027 |   2028 |   2029 |   2030 |   2021-2030 |
|:--------|--------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|-------:|------------:|
| IIT     | -268.98 |      -23.39 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |     -292.37 |
| Payroll |    0    |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |      0 |        0    |

**Comment by jdebacker** [comment]
@amshoulders After you merge my PR to your branch (with updates to the expected files when the model is run with the PUF), I'll give this PR a final review.  Thanks!

**Comment by jdebacker** [comment]
I've left a couple small comments, but otherwise this PR looks good to me.  Thanks @amshoulders!

**Comment by amshoulders** [comment]
This is the revenue estimate from the cps file:


2020 | 2021 | 2022 | 2023 | 2024 | 2025 | 2026 | 2027 | 2028 | 2029 | 2030 | 2021-2030
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
IIT | -353.37 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | -353.37
Payroll | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0

I also made the changes that @jdebacker pointed out.  Everything should be good to go on this PR!

**Comment by jdebacker** [comment]
@amshoulders Thanks for this PR!  LGTM.  I'll leave open for a few more hours in case @MattHJensen or others want to comment before merging.

**Pull request merged**